packages/crud/src/vaadin-dialog-layout.js

```diff
@@ -72,29 +72,29 @@ class DialogLayout extends ElementMixin(ThemableMixin(PolymerElement)) {
         }
       </style>
 
-      <div id="editor" part="editor" hidden$="[[!_computeEditorOpened(opened, mobile, 'bottom','aside')]]">
+      <div id="editor" part="editor" hidden$="[[__computeEditorHidden(opened, mobile, editorPosition)]]">
         <div part="scroller" id="scroller" role="group" aria-labelledby="header">
           <div part="header" id="header">
             <slot name="header"></slot>
           </div>
-          <slot></slot>
+          <slot name="form"></slot>
         </div>
 
         <div part="footer" role="toolbar">
-          <slot name="footer"></slot>
+          <slot name="save-button"></slot>
+          <slot name="cancel-button"></slot>
+          <slot name="delete-button"></slot>
         </div>
       </div>
 
-      <vaadin-dialog
+      <vaadin-crud-dialog
         id="dialog"
-        opened="[[_computeEditorOpened(opened, mobile, '')]]"
+        opened="[[__computeDialogOpened(opened, mobile, editorPosition)]]"
         aria-label="[[__ariaLabel]]"
         no-close-on-outside-click="[[noCloseOnOutsideClick]]"
         no-close-on-esc="[[noCloseOnEsc]]"
         theme$="[[theme]] layout"
```

Comment:
Seems that applying the `no-border` theme to crud activates the dialog's `no-border` theme:
![Screenshot 2021-11-25 at 12 15 05](https://user-images.githubusercontent.com/1222264/143423334-5cfe0fa6-0bdc-4066-909c-8acfd138f8de.png)

Reply:
I don't think Dialog has a `no-border` variant, but `no-padding` for sure.

Reply:
Ah, true. Then it's something else

Reply:
Fixed by changing `layout` to `crud` here and in the corresponding styles.